packages/core/src/components/Typography/FontWeight/FontWeightSection.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box } from '@chakra-ui/react'
+import { TFontSizeVariant, TFontWeightVariant } from '@core/types'
+
+import { DisplayFontProperty } from '../DisplayFontProperty'
+
+export function FontWeightSection({
+  fontWeightData,
+  onUpdateFontPropertyData,
+}: {
+  fontWeightData: TFontWeightVariant[]
+  onUpdateFontPropertyData: (
+    newFontWeightData: TFontSizeVariant[] | TFontWeightVariant[]
+  ) => void
+}) {
+  const headingText = 'Font Weights'
+  const buttonText = 'Add New Font Weight'
```

Comment:
nit: No need to redefine these extra variables here, I'd recommend just passing in a string below. IMO makes the code a tiny bit harder to follow because I now have to go trace what this variable is from below.